syntax = "proto3";

message Empty {};

message BlocksRequest {
  uint64 start_block_number = 2;
  optional uint64 stop_block_number = 3;
};

message HeightMessage {
    uint64 height = 1;
};
message PeersMessage {
    repeated Peer peers = 1;
};
message Peer {
    string address = 1;
    uint64 height = 2;
    uint64 last_seen = 3;
};

message MempoolSizeMessage {
    uint64 size = 1;
};
